### DISCUSSION TITLE
Home Screen for Messaging

### DISCUSSION THREAD

**Discussion by esethna** [mattermost/community#62] [Practical Ideas]
Similar to how Boards and Playbooks will have a home screen, messaging should have one too. We could implement some intelligence to provide easy access to things that the user may care about. 

- Seeing what channels have active meetings ongoing with the ability to join them
- Smart activity monitoring if there's active channels: "There's a lot of conversation happening in channel y, check it out"
- Smart channel discovery "your collegues are in channel z, join it?" "There's a lot of activity in this channel you might be missing, join it" 
- Digest of unreads that you missed since last activity

**Comment by enahum** [comment]
@michaelgamble do you have the mocks we did during the lisbon hackathon?

**Reply by esethna** [reply]
Oh I didn't know we had design ideas on this, please share!

**Comment by chenilim** [comment]
+100 this. The most important feature for me personally is the second point above - being able to discover new and popular channels. I used to browse the public channel list every few months, join all the new channels, then leave the empty or abandoned ones.

Even showing an indication of channel activity (e.g. last post X days ago, Y members) would be super helpful.

**Reply by esethna** [reply]
Ya totally, I think there is probably some easy algorithms to determine what might be relevant or popular to specific users

**Comment by chetanyakan** [comment]
This can also be implemented by creating a `discover` channel and a bot that posts messages in the discover channel.

OR a discover bot that DMs you the channel list personalized for you.

**Reply by esethna** [reply]
Yep definitely many mechanisms to achieve the desired outcome of channel discoverability
